bottlerocket/testsys/src/run_aws_ecs.rs

```diff
@@ -355,7 +355,10 @@ impl RunAwsEcs {
             node_ami: self.ami.clone(),
             // TODO - configurable
             instance_count: Some(2),
-            instance_type: self.instance_type.clone(),
+            instance_types: match self.instance_type.clone() {
+                Some(instance_type) => vec![instance_type],
+                None => vec![],
+            },
```

Comment:
This may be cleaner: `self.instance_type.clone().iter().collect()` or even `self.instance_type.map(|instance_type| vec![instance_type.clone()]).unwrap_or_default()`.